lib/suspenders/generators/app_generator.rb

```diff
@@ -160,31 +159,29 @@ def install_bitters
     end
 
     def install_refills
-      say "Install Refills"
+      say 'Install Refills'
       build :install_refills
     end
 
     def setup_git
-      if !options[:skip_git]
-        say 'Initializing git'
-        invoke :setup_gitignore
-        invoke :init_git
-      end
+      return if options[:skip_git]
+      say 'Initializing git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.